ckanext/dcat/logic.py

```diff
@@ -30,6 +29,21 @@ def dcat_dataset_show(context, data_dict):
 
     return output
 
+def dcat_organization_show(context, data_dict):
+
+    toolkit.check_access('dcat_organization_show', context, data_dict)
+
+    query = _search_ckan_datasets(context, data_dict)
```

Comment:
Add the `fq` parameter here, that makes the API for the action cleaner (you just pass the `id` of the organization).
Also use this as the value for `fq`, to allow using the name (what's on the URL) and the internal id:

```
'organization:{id} OR owner_org:{id}'.format(id=_id)
```